src/nbl/video/ILogicalDevice.cpp

```diff
@@ -420,6 +420,14 @@ bool ILogicalDevice::updateDescriptorSets(const std::span<const IGPUDescriptorSe
     return true;
 }
 
+void ILogicalDevice::nullifyDescriptors(const std::span<const IGPUDescriptorSet::SDropDescriptorSet> dropDescriptors)
+{
+    for (const auto& drop : dropDescriptors)
```

Comment:
you need the same validation that `updateDescriptorSets` has for the writes, insofar as its applicable

Function should also return a bool of whether we've had success or not